.github/workflows/test.yml

```
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        pgversion: [16, 15, 14, 13, 12]
+        pgversion: [18, 17, 16, 15, 14, 13, 12]
```

```suggestion
        pgversion: [17, 16, 15, 14, 13, 12]
```
Test 17 only, encountered a CI error on GRANT on Pg18